### ISSUE TITLE
Update to 2021.12 showing a warning that asks to report to integration developer

### ISSUE THREAD

**Issue by user897943** [lolouk44/CurrentCost_HA_CC#15]
---
name: Bug report
about: Create a report to help us improve
title: ''
labels: bug
assignees: ''

---

**Describe the bug**

```
Logger: homeassistant.helpers.entity
Source: helpers/entity.py:549
First occurred: 6:02:51 PM (3 occurrences)
Last logged: 6:03:21 PM

Entity sensor.current_cost (<class 'custom_components.currentcost.sensor.CurrentCostSensor'>) implements device_state_attributes. Please report it to the custom component author.
```

**To Reproduce**
Steps to reproduce the behaviour, including error message if any.

Install from HACS. Upgrade HA to 2021.12, this appears on the log.

**Expected behaviour**

No warnings from HA on the log. If HA is being explicit by asking to contact the developer then there must be a reason for this to be fixed

**Screenshots**

**Desktop/Server/Device (please complete the following information):**
 - Device used to run the Custom Component [e.g. Raspberry Pi, NUC]
 - Method of installation [e.g. via HACS or manual]
 - Version of the Custom Component installed?
 - Model of the Current Cost Device

**Additional context**

**Logs**
Please set the Current Cost Custom Component's logging level to debug and provide relevant logs section from Home-Assistant.
To Enable debug logging level, add this to your configuration.yaml and restart:

```
logger:
  default: error
  logs:
    custom_components.currentcost: debug
```

**Comment by lolouk44** [comment]
Thanks. I'll have a look this weekend

**Comment by user897943** [comment]
Thank you @lolouk44

**Comment by lolouk44** [comment]
@user897943 
I had a look at the code and changes appear trivial, however I no longer use my Current Cost so can't test it.
 Would you be able to test a beta version?

**Comment by user897943** [comment]
Sure, no problem, how? Will you push beta to HACS? Thanks for maintaining
this despite stopping to use it.

**Comment by lolouk44** [comment]
Great, thanks for your help.

There should be just one thing to change:
In `[config folder]\custom_components\currentcost\sensor.py`, line 193, replace `device_state_attributes` with `extra_state_attributes`
sensor.py 
If that works I'll update the published code and will push an update via HACS

**Comment by timutta** [comment]
I can confirm that it works, I did change the same entry on my local sensor.py.

**Comment by lolouk44** [comment]
Thanks, fixed in [0.2.2](https://github.com/lolouk44/CurrentCost_HA_CC/releases/tag/0.2.2)
